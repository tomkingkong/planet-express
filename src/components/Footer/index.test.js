import React from 'react';
import { shallow } from 'enzyme';

import { Footer } from '.';

describe('Footer component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });

