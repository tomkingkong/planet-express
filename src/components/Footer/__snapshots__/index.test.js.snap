// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<div
  className="Footer"
>
  <Link
    className="profile__btn"
    replace={false}
    to="/profile"
  >
    Profile
  </Link>
</div>
`;

exports[`Footer component should match snapshot with backbutton 1`] = `
<div
  className="Footer"
>
  <button
    className="footer__back"
    onClick={[Function]}
  >
    BACK
  </button>
  <Link
    className="profile__btn"
    replace={false}
    to="/profile"
  >
    Profile
  </Link>
</div>
`;
